joomla/joomla-cms
# [com_templates] Add prefix to class name in menu assignment

Pull Request for Issue #24605.

### Summary of Changes
Add prefix to class name to prevent conflicting with existing classes.


### Testing Instructions
Create a new Menu named Hidden and set the menutype to "hidden".
Add a menu item to that menu.
Open a template style and check the Menu Assignment tab.


### Expected result
A visible checkbox to associate the template style with the created menu item.


### Actual result
No checkbox visible. Reason for that is the menu alias will be added to the Input tag as a class - without any prefix. So it becomes input[type=checkbox].hidden which is a class used by bootstrap to hide stuff.
